resources/rest-service/cloudify/migrations/versions/423a1643f365_4_6_to_5_0.py

```diff
@@ -22,28 +26,181 @@
 depends_on = None
 
 
+Base = declarative_base()
+
+
+class Config(Base):
```

Comment:
the model is also written in the migration scrip, so that we can use the ORM to insert values in the migration